src/components/assets/AssetCard.vue

```diff
@@ -68,11 +68,15 @@ export default {
     }
   },
   data: () => ({
-    imgNotFound: require('@/assets/img-not-found.png')
+    imgNotFound: require('@/assets/img-not-found.png'),
+    thumbnailUrl: undefined
   }),
+  created() {
+    this.thumbnailUrl = this.asset.thumbnailUrl
+  },
   methods: {
-    imgError(asset) {
-      asset.thumbnailUrl = this.imgNotFound
+    imgError() {
+      this.thumbnailUrl = this.imgNotFound
```

Comment:
Modifying prop `asset` attribute was causing error.

Reply:
Ah. I think I can tweak such that `thumbnailUrl` is initialized in `data: () => ...` using asset prop.